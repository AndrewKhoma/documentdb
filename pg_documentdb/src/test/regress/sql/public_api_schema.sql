-- show all functions exported in documentdb_api.
\df documentdb_api.*

\df documentdb_api_catalog.*

\df documentdb_data.*

-- show all aggregates exported
\da+ documentdb_api.*

\da+ documentdb_api_catalog.*

\da+ documentdb_data.*

-- Access methods + Operator families
\dA *documentdb*

\dAc *documentdb*

\dAf *documentdb*

\dX *documentdb*

-- This is last (Tables/indexes)
\d documentdb_api.*

\d documentdb_api_catalog.*
