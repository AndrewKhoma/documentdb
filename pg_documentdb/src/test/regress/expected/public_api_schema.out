-- show all functions exported in documentdb_api.
\df documentdb_api.*
                       List of functions
 Schema | Name | Result data type | Argument data types | Type 
--------+------+------------------+---------------------+------
(0 rows)

\df documentdb_api_catalog.*
                       List of functions
 Schema | Name | Result data type | Argument data types | Type 
--------+------+------------------+---------------------+------
(0 rows)

\df documentdb_data.*
                       List of functions
 Schema | Name | Result data type | Argument data types | Type 
--------+------+------------------+---------------------+------
(0 rows)

-- show all aggregates exported
\da+ documentdb_api.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
--------+------+------------------+---------------------+-------------
(0 rows)

\da+ documentdb_api_catalog.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
--------+------+------------------+---------------------+-------------
(0 rows)

\da+ documentdb_data.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
--------+------+------------------+---------------------+-------------
(0 rows)

-- Access methods + Operator families
\dA *documentdb*
List of access methods
 Name | Type 
------+------
(0 rows)

\dAc *documentdb*
                  List of operator classes
 AM | Input type | Storage type | Operator class | Default? 
----+------------+--------------+----------------+----------
(0 rows)

\dAf *documentdb*
        List of operator families
 AM | Operator family | Applicable types 
----+-----------------+------------------
(0 rows)

\dX *documentdb*
                 List of extended statistics
 Schema | Name | Definition | Ndistinct | Dependencies | MCV 
--------+------+------------+-----------+--------------+-----
(0 rows)

-- This is last (Tables/indexes)
\d documentdb_api.*
\d documentdb_api_catalog.*
